src/Seq.js

```diff
@@ -192,7 +192,9 @@ export class ArraySeq extends IndexedSeq {
 
 class ObjectSeq extends KeyedSeq {
   constructor(object) {
-    const keys = Object.keys(object);
+    const keys = Object.keys(object).concat(
+      Object.getOwnPropertySymbols(object)
+    );
```

Comment:
This is a relatively hot path, could you do a quick and dirty performance test to ensure this won't cause a regression?

Also, this library is used in broad-support applications, this will likely throw an error if run on Internet Explorer or older browsers.

Reply:
Absolutely. Good call.

If `getOwnPropertySymbols` doesn't work in IE or older browser then I'll probably want to check for the existence/usability of that method, correct?

Is it possible to run the test suite in Travis on IE, somehow?

Reply:
> I'll probably want to check for the existence/usability of that method, correct?

Yep!

> Is it possible to run the test suite in Travis on IE, somehow?

Not that I'm aware of.

Reply:
Oh, I think I may have misunderstood. It looks like there are already some performance tests in this repo. `perf/Map.js` looks like the correct test to run, but I don't think it is run by `npm run perf` (which i'm struggling to get running anyway). Am I on the right track? Were you looking for new tests?

Reply:
More specifically, i get this error:
```bash
21:51 $ npm run perf

> immutable@4.0.0-rc.7 perf /Users/abacaphiliac/github.com/facebook/immutable-js
> node ./resources/bench.js

ugh Error: Command failed: git show master:dist/immutable.js
fatal: Path 'dist/immutable.js' exists on disk, but not in 'master'.
```

I think I must have missed a step.

Reply:
I think I got it! I had to build and commit `dist` in master to get the perf script to run on my branch. I am so sorry about all the mail...

### Before
[before.txt](https://github.com/facebook/immutable-js/files/1389872/before.txt)

### After
[after.txt](https://github.com/facebook/immutable-js/files/1389873/after.txt)

To the untrained eye, it looks like a lot of thrashing in my branch. What are your thoughts?